### DISCUSSION TITLE
Demos

### DISCUSSION THREAD

**Discussion by josu98** [oscal-compass/compliance-trestle#1532] [General] [answered]
Hi,
There are a bunch of links to demos but they doesnt work??
Do you have any working demos?
Regards

**Comment by degenaro** [comment, marked as answer]
Please try again. The link comprising
https://github.com/oscal-compass/compliance-trestle-demos should be fixed.
Any others?

Lou.

**Comment by AleJo2995** [comment]
Hi @josu98 did you get the demos working? Thanks & Regards

**Reply by josu98** [reply]
Hi!
Yes, thanks.
Regards

**Comment by imperialstout1** [comment]
I was attempting to go through the author ssp demo and I get an error on the trestle add command. That seems I don't have the add subcommand. 

trestle: error: argument cmd: invalid choice: 'add' (choose from 'assemble', 'author', 'create', 'describe', 'href', 'import', 'init', 'merge', 'partial-object-validate', 'remove', 'replicate', 'split', 'task', 'validate', 'version')

**Reply by imperialstout1** [reply]
Ah, I finally found that 'add' was merged with 'create'. Demo docs are out of date.

**Comment by degenaro** [comment]
Very nice! Do you want to open an issue to fix the doc? Please be specific
on what the where in the doc the problem lies.

And, if you want to contribute the fixed doc as a PR that would be
wonderful (but not required).

**Reply by imperialstout1** [reply]
I can open an issue for it, though I'm still working through the demo personally. I was attempting it with the vanilla catalog/profiles from NIST's repo and ran into an issue with the parameter references not matching.

**Reply by degenaro** [reply]
Thx for opening https://github.com/oscal-compass/compliance-trestle/issues/1666. If you find additional related problems, you can add them to this issue. If you'd like to create a PR to fix we can assign the issue to you. Next community meeting is Tue. Aug. 27 at 11:00 ET https://github.com/oscal-compass/community/blob/main/README.md. We can add this issue to the agenda for a brief discussion.

**Reply by imperialstout1** [reply]
My focus on the project I am working was just reassigned to a higher priority item, so I won't be able to do a PR for it. I might be re-looking at OSCAL in a few months.
